TeamCode/src/main/java/org/firstinspires/ftc/teamcode/systems/drive/MecanumDriveSystem.java

```diff
@@ -145,11 +158,20 @@ public void driveGodMode(float rightX, float rightY, float leftX, float leftY, f
         motorBackRight.setPower(backRight);
     }
 
-    /**
-     * Sets motor values based on the joystick x and y values.
-     * @param x X value of the joystick
-     * @param y Y value of the joystick
-     */
+    private void clampPowers(List<Double> powers) {
```

Comment:
Remove clampPowers from code

Reply:
I left that in there because I thought it was important for godMode for some reason. happy to see it go